src/store/AnnotationsAPI.ts

```diff
@@ -69,14 +69,11 @@ export default class AnnotationsAPI {
       });
   }
 
-  async getAnnotationsForDatasetId(id: string): Promise<IAnnotation[]> {
+  async getAnnotationsForDatasetId(datasetId: string): Promise<IAnnotation[]> {
     const annotations: IAnnotation[] = [];
-    const pages = await fetchAllPages(
-      this.client,
-      "upenn_annotation",
-      id,
-      100000
-    );
+    const pages = await fetchAllPages(this.client, "upenn_annotation", {
+      params: { datasetId, limit: 100000, sort: "_id" }
```

Comment:
The limit value is the limit of documents per api call
There already is a default limit of 1000 in `fetchAllPages`
It is only at 1000 to avoid fetching to much data at the same time
The limit is adapted by the caller depending on the things that are fetched
Here, for annotations, a limit of 100000 is fine but it depends only on the size of the documents fetched

Reply:
Oh ok, maybe I should instead increase the default limit to 100000 and not specify anything when calling `fetchAllPages`